runtime/src/validate_block/implementation.rs

```diff
@@ -220,6 +233,24 @@ impl<B: BlockT> Storage for WitnessStorage<B> {
 			self.overlay.insert(key, None);
 		}
 	}
+
+	fn storage_append(&mut self, key: &[u8], value: Vec<u8>) {
+		let key_vec = key.to_vec();
+		let mut value_vec = Vec::with_capacity(1);
+		value_vec.push(EncodeOpaqueValue(value));
```

Comment:
```suggestion
		let value_vec = vec![EncodeOpaqueValue(value)];
```

Reply:
Not available in nostd. Or am I missing something?

Reply:
`sp_std::vec!`

Reply:
ok fixed